src/Routers/SchemasRouter.js

```diff
@@ -6,6 +6,15 @@ var Parse = require('parse/node').Parse,
 import PromiseRouter from '../PromiseRouter';
 import * as middleware from '../middlewares';
 
+const checkSchemasOption = req => {
+  if (req.config.schemas) {
+    throw new Parse.Error(
+      Parse.Error.OPERATION_FORBIDDEN,
+      'This server use schemas option, schemas creation/modification operations are blocked. You can still delete a class.'
```

Comment:
'This server uses schemas option. Schemas creation/modification operations are blocked. Classes can still be deleted.'